public/bitemp_graph.js

```diff
@@ -526,7 +526,7 @@ var barChart = function() {
         })
         .text(function(d) {
           var str = '';
-          if(window.location.href.endsWith('/search')) {
+          if(window.location.href.endsWith('/')) {
```

Comment:
I think "window.location.pathname === '/'" would be a better test here. That avoids the possibility of someone typing an extra slash and future-proofs against using URL parameters at some point (which are included in .href but not in .pathname).